### PR TITLE
Fixed semver validation on PCI import.

### PR DESCRIPTION
**Pull request by ekkinox**

### CODE REVIEW

File: model/portableElement/validator/Validator.php [hunk 1]
```diff
@@ -184,6 +186,23 @@ public static function isValidVersion($value)
...
+    /**
+     * @param $value
+     * @return bool
+     * @throws PortableElementInvalidFieldException
+     */
+    public static function isValidSemVer($value)
+    {
+        $validator = \tao_helpers_form_FormFactory::getValidator(
+            self::Regex,
+            array('format' => '/^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?(\+[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?$/')
```
Comment (ssipasseuth) [inline model/portableElement/validator/Validator.php]: We are using this lib already: 
https://github.com/oat-sa/extension-tao-itemqti/blob/a4e4026602c78f22e99ce9a6a2c722ac180be794/composer.json#L61

Would it be possible to leverage it by reusing the helper https://github.com/naneau/semver/blob/master/src/Naneau/SemVer/Regex.php#L45-L66 ?

Comment (ekkinox) [reply model/portableElement/validator/Validator.php]: done

Comment (ssipasseuth) [reply model/portableElement/validator/Validator.php]: Ok, looks good :)

File: test/unit/model/portableElement/validator/ValidatorTest.php [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+namespace oat\taoQtiItem\test\unit\model\portableElement\validator;
+
+use oat\tao\test\TaoPhpUnitTestRunner;
+use oat\taoQtiItem\model\portableElement\element\PortableElementObject;
+use oat\taoQtiItem\model\portableElement\exception\PortableElementInvalidFieldException;
+use oat\taoQtiItem\model\portableElement\exception\PortableElementInvalidModelException;
+use oat\taoQtiItem\model\portableElement\validator\Validatable;
+use oat\taoQtiItem\model\portableElement\validator\Validator;
+
+class ValidatorTest extends TaoPhpUnitTestRunner
```
Comment (siwane) [inline test/unit/model/portableElement/validator/ValidatorTest.php]: This class is deprecated use TestCase instead please

Comment (ekkinox) [reply test/unit/model/portableElement/validator/ValidatorTest.php]: fixed on both tests

**Pull request merged**